res/locales/it/modules.go

```diff
@@ -0,0 +1,239 @@
+			},
+			Replacer: modules.MovieSearchReplacer,
+		},
+
+		{
+			Tag: modules.MoviesDataTag,
+			Patterns: []string{
+				"I'm bored",
+				"I don't know what to do",
+			},
+			Responses: []string{
+				"I propose you a movie of %s “%s” which is rated %.02f/5",
+			},
+			Replacer: modules.MovieSearchFromInformationReplacer,
+		},
+
+		// NAME
+		{
+			Tag: modules.NameGetterTag,
+			Patterns: []string{
+				"Do you know my name?",
+			},
+			Responses: []string{
+				"Your name is %s!",
+			},
+			Replacer: modules.NameGetterReplacer,
+		},
+
+		{
+			Tag: modules.NameSetterTag,
+			Patterns: []string{
+				"My name is ",
+				"You can call me ",
+			},
+			Responses: []string{
+				"Great! Hi %s",
+			},
+			Replacer: modules.NameSetterReplacer,
+		},
+
+		// RANDOM
+		{
+			Tag: modules.RandomTag,
+			Patterns: []string{
+				"Give me a random number",
+				"Generate a random number",
+			},
+			Responses: []string{
+				"The number is %s",
+			},
+			Replacer: modules.RandomNumberReplacer,
+		},
+
+		// REMINDERS
+		// Translations are required in `language/date/date`, `language/date/rules` and in `language/reason`,
+		// please don't forget to translate it.
+		// Otherwise, remove the registration of the Reminders modules in this file.
+
+		{
+			Tag: modules.ReminderSetterTag,
+			Patterns: []string{
+				"Remind me to cook a breakfast at 8pm",
+				"Remind me to call mom tuesday",
+				"Note that I have an exam",
+				"Remind me that I have a conference call tomorrow at 9pm",
+			},
+			Responses: []string{
+				"Noted! I will remind you: “%s” for the %s",
+			},
+			Replacer: modules.ReminderSetterReplacer,
+		},
+
+		{
+			Tag: modules.ReminderGetterTag,
+			Patterns: []string{
+				"What did I ask for you to remember",
+				"Give me my reminders",
+			},
+			Responses: []string{
+				"Mi hai chiesto di ricordarti queste cose:\n%s",
+			},
+			Replacer: modules.ReminderGetterReplacer,
+		},
+
+		// SPOTIFY
+		// A translation is needed in `language/music`, please don't forget to translate it.
+		// Otherwise, remove the registration of the Spotify modules in this file.
+
+		{
+			Tag: modules.SpotifySetterTag,
+			Patterns: []string{
+				"I miei token di spotify",
+				"I miei segreti spotify",
+			},
+			Responses: []string{
+				"Login in corso",
+			},
+			Replacer: modules.SpotifySetterReplacer,
+		},
+
+		{
+			Tag: modules.SpotifyPlayerTag,
+			Patterns: []string{
+				"Riproduci su Spotify",
+			},
+			Responses: []string{
+				"In riproduzione %s da %s su Spotify.",
+			},
+			Replacer: modules.SpotifyPlayerReplacer,
+		},
+	})
+
+	// COUNTRIES
+	// Please translate this method for adding the correct article in front of countries names.
+	// Otherwise, remove the countries modules from this file.
+
+	modules.ArticleCountries["it"] = ArticleCountries
+}
+
+// ArticleCountries returns the country with its article in front.
+func ArticleCountries(name string) string {
```

Comment:
You should either create a method that finds the article of a country or remove it

Reply:
that file is  a WIP. I have copied the en training.json and translated it

Reply:
Okay, good luck!